### DISCUSSION TITLE
iOS中不支持xib中使用PAGView 即 通过initWithCoder初始化的PAGView无法播放资源

### DISCUSSION THREAD

**Discussion by AshineReal** [Tencent/libpag#1560] [General] [answered]
如题 不支持解档的方式 在xib中加载PAGView ，目前只能通过init初始化的方式才能正常播放资源

**Comment by FusionXu** [comment]
可以通过附件提供一下demo，在你提供的demo上才能更好地定位问题

**Reply by wangwanjie** [reply]
谁好人用啥xib啊

**Reply by kevingpqi123** [reply, marked as answer]
PAGView 源码中仅仅实现了 initWithFrame 没有实现 initWithCoder， 可以继承 PAGView 复写 initWithCoder 调用 PAGView 的 initWithFrame 就可以了

**Reply by AshineReal** [reply]
嗯 只有通过继承或者包裹一层 来解决
